client/src/Organisms/DogData/DogData.js

```diff
@@ -21,8 +20,19 @@ const DogData = ({ id }) => {
   const { contestState } = useContext(ContestContext);
 
   useEffect(() => {
-    setDogData(doggos.find((dog) => dog.dogId === id));
-    setIsPending(false);
+    const requestOptions = {
+      method: 'GET',
+      redirect: 'follow',
+      credentials: 'include',
+    };
+
+    fetch(`http://localhost:27020/api/dogs/${id}`, requestOptions)
+      .then((response) => response.json())
+      .then((result) => {
```

Comment:
jakby nie bylo resultsow, to albo mozna gdzies przekierowac, a jezeli po prostu ni ma psow - napisac ze jeszcze nie masz zadnego psa